hudi-common/src/main/java/org/apache/hudi/common/model/HoodieTimelineTimeZone.java

```diff
@@ -18,20 +18,29 @@
 
 package org.apache.hudi.common.model;
 
+import java.time.ZoneId;
+import java.util.TimeZone;
+
 /**
  * Hoodie TimelineZone.
  */
 public enum HoodieTimelineTimeZone {
-  LOCAL("local"),
-  UTC("utc");
+  LOCAL("local", ZoneId.systemDefault()),
+  UTC("utc", TimeZone.getTimeZone("UTC").toZoneId());
 
   private final String timeZone;
+  private final ZoneId zoneId;
```

Comment:
Don't think we need a zoneId member, can we calculate it on the fly in method `getZoneId` ?

Reply:
I can do a switch-case but I feel this is better. WDYT?

Reply:
Fine, it's just a preference.